website/src/css/_headers.scss

```diff
@@ -15,7 +15,17 @@ h3 {
   font-size: var(--ifm-h3-font-size);
 }
 
-p {
+p,
+li,
+a {
   color: var(--p-color);
-  font-size: 0.875rem;
+  font-size: var(--font-size);
+}
+
+@media (min-width: 1000px) {
```

Comment:
I don' like to have this breakpoint like this, do you know if there are exposed breakpoints?